fishtank/src/node.ts

```diff
@@ -116,13 +116,36 @@ export class Node {
     throw new Error(`Timeout of ${timeout}ms exceeded`)
   }
 
-  async mineUntil(until: { blockSequence: number }): Promise<void> {
+  async mineUntil(
+    until: { blockSequence: number } | { transactionMined: string },
```

Comment:
until can only be blockSequence or transactionMined, we might have use case that need to meet both condition, eg mine with a transaction and until pass the activation sequence.

Reply:
You can just call `mineUntil` twice--but yeah, we can think about extending this to support some AND/OR conditions